ang/crmUi.js

```diff
@@ -14,19 +14,18 @@
         scope: {
           crmUiAccordion: '='
         },
-        template: '<div ng-class="cssClasses"><div class="crm-accordion-header">{{crmUiAccordion.title}} <a crm-ui-help="help" ng-if="help"></a></div><div class="crm-accordion-body" ng-transclude></div></div>',
+        template: '<details class="crm-accordion-wrapper"><summary class="crm-accordion-header">{{crmUiAccordion.title}} <a crm-ui-help="help" ng-if="help"></a></summary><div class="crm-accordion-body" ng-transclude></div></details>',
         transclude: true,
         link: function (scope, element, attrs) {
-          scope.cssClasses = {
-            'crm-accordion-wrapper': true,
-            collapsed: scope.crmUiAccordion.collapsed
-          };
           scope.help = null;
           scope.$watch('crmUiAccordion', function(crmUiAccordion) {
-            if (crmUiAccordion && crmUiAccordion.help) {
-              scope.help = crmUiAccordion.help.clone({}, {
-                title: crmUiAccordion.title
-              });
+            if (crmUiAccordion) {
+              $(element).children('details').prop('open', !crmUiAccordion.collapsed);
```

Comment:
For `r-run`, I tried the "New Mailing" screen and focused on the behavior of the `collapsed: {BOOL}` option. It did the main things that I expected.

There was one quirk, as seen by:

1. Create a new mailing
2. Open the "Plain Text" section
3. Type "Abc"
4. Press backspace a few times
5. The whole accordion auto-collapses.

It feels kinda unstable. OTOH, I think it might be just as well to start converting the `<div crm-ui-accordion>`s to literal`<details>`, in which case it wouldn't matter too much if it's quirky.

Reply:
@totten ok I've pushed a fix to that instability issue and also added a `@deprecated` comment because there's no reason for new code to use this directive.